english/marshmallow.md

```diff
@@ -86,4 +86,6 @@ Do you:
 
 [Remember that pill you took](adipose/adipose.md)
 
-[light up the wall](light-up/light-up.md)
\ No newline at end of file
+[light up the wall](light-up/light-up.md)
+
+[hide the lighter fork and rope in your pocket](hide/hide.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.